privaterelay/templates/includes/modal-domain-registration-confirmation.html

```diff
@@ -24,12 +24,35 @@ <h3><strong class="js-modal-domain-registration-confirmation-domain-preview"></s
     </div>
 
     <div class="c-modal-footer">
+      <button type="submit" disabled class="mzp-c-button mzp-t-product js-modal-domain-registration-submit">
+        {% ftlmsg 'modal-domain-register-button' %}
+      </button>
       <button type="button" class="mzp-c-button mzp-t-neutral js-modal-domain-registration-cancel">
         {% ftlmsg 'profile-label-cancel' %}
       </button>
-      <button type="submit" disabled class="mzp-c-button mzp-t-product js-modal-domain-registration-submit">
-        {% ftlmsg 'modal-domain-register-button' %}
+    </div>
+  </div>
+
+  <div class="c-modal is-hidden t-domain-registration-success js-domain-registration-success">
```

Comment:
This is the "success" state that appears during the onboarding registration process: 
<img width="511" alt="image" src="https://user-images.githubusercontent.com/2692333/140422850-e58ad97b-05cb-43a9-921c-99f76898253d.png">

_(Note that this is not shown currently if you register from the dashboard/profile page)_